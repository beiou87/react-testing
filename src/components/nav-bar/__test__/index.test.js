import React from 'react';
import renderer from 'react-test-renderer';
import NavBar from '..';

it('renders correctly', () => {
  const tree = renderer.create(
    <NavBar
      leftContent={(<button>back</button>)}
      leftContent={(<button>save</button>)}      
      >
      Hello
    </NavBar>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
