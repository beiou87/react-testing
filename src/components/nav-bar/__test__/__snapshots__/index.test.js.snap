// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="nav-bar"
>
  <div
    className="nav-bar-left"
  >
    <button>
      save
    </button>
  </div>
  <div
    className="nav-bar-title"
  >
    Hello
  </div>
  <div
    className="nav-bar-right"
  />
</div>
`;
